.github/workflows/python-tox.yml

```diff
@@ -5,57 +5,20 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
-    services:
```

Comment:
I'd already changed the test settings so that sqlite in-memory is used. We don't test any specific database functionality.